lldb/include/lldb/Target/Target.h

```diff
@@ -1317,6 +1317,7 @@ class Target : public std::enable_shared_from_this<Target>,
     enum class StopHookResult : uint32_t {
       KeepStopped = 0,
       RequestContinue,
+      NoPreference,
```

Comment:
Happy to bike shed this name.  The difference between this and `RequestContinue` is that we keep the default stance of "stop" if there are only "no preference" but no other hooks.